### DISCUSSION TITLE
Exposure time

### DISCUSSION THREAD

**Discussion by mshetzer** [aaronwmorris/indi-allsky#1058] [General]
Aaron, love the app !!!  
I'm using an old ZWO224 MC, which gain stops at 60.  Is there any way I can get a longer exposure time than 30 seconds?  I don't care if the stars are not round, but I'm just not getting enough data for the charts to work.

If not, is there a reasonable priced camera you recommend?  Preferable same form factor as I built a 3d mount for it.

Thanks for all your hard work!
matt

**Comment by aaronwmorris** [comment]
Thank you very much!

That does not sound right regarding the ASI224MC.  The gain for the ASI224MC should go up to at least 300 and the max exposure should go as high as 1000s.

You can see Gord Tulloch's ASI224MC is using gain 100 here:  https://www.gordtulloch.com/indi-allsky
